be/test/storage/conjunctive_predicates_test.cpp

```diff
@@ -23,6 +24,22 @@
 
 namespace starrocks::vectorized {
 
+inline TExprOpcode::type convert_predicate_type_to_thrift(PredicateType p) {
+    static std::unordered_map<PredicateType, TExprOpcode::type> mapping = {
+            {PredicateType::kEQ, TExprOpcode::EQ},
+            {PredicateType::kNE, TExprOpcode::NE},
+            {PredicateType::kGT, TExprOpcode::GT},
+            {PredicateType::kGE, TExprOpcode::GE},
+            {PredicateType::kLT, TExprOpcode::LT},
+            {PredicateType::kLE, TExprOpcode::LE},
+            {PredicateType::kInList, TExprOpcode::FILTER_IN},
+            {PredicateType::kNotInList, TExprOpcode::FILTER_NOT_IN},
+            {PredicateType::kAnd, TExprOpcode::COMPOUND_AND},
+            {PredicateType::kOr, TExprOpcode::COMPOUND_OR},
+    };
+    return mapping[p];
```

Comment:
If the input PredicateType is not included in the mapping. what will happen? And is that what you want?